### ISSUE TITLE
filename error (?) when running from command line

### ISSUE THREAD

**Issue by schluppeck** [elliohow/fMRI_ROI_Analysis_Tool#1]
... switched off multicore processing and tried running on `Test_dataset`

- FSL version / python

```bash
cat  /usr/local/fsl/etc/fslversion
## 6.0.3:b862cdd5

python3 --version
## Python 3.7.7
```

here some more details

```bash
lpzds1 (master *) fRAT $ python3 fRAT_GUI.py
```

DEPRECATION WARNING: The system version of Tk is deprecated and may be removed in a future release. Please don't rely on it. Set TK_SILENCE_DEPRECATION=1 to suppress this warning.
--- Running fRAT ---
--- Saved settings ---

--- Environment Setup ---
Select the directory of the raw MRI/fMRI brains.
Selected directory: /Users/lpzds1/Desktop/tidy/Test_dataset
Using the HarvardOxford-Cortical atlas.

--- Analysis ---

Analysing brain 1/6: 1901_WIPMB4TASKfMRI_singleechoTR2_e1.nii.

Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/nipype/interfaces/base/core.py", line 522, in aggregate_outputs
    setattr(outputs, key, val)
  File "/usr/local/lib/python3.7/site-packages/nipype/interfaces/base/traits_extension.py", line 330, in validate
    value = super(File, self).validate(objekt, name, value, return_pathlike=True)
  File "/usr/local/lib/python3.7/site-packages/nipype/interfaces/base/traits_extension.py", line 135, in validate
    self.error(objekt, name, str(value))
  File "/usr/local/lib/python3.7/site-packages/traits/base_trait_handler.py", line 77, in error
    object, name, self.full_info(object, name, value), value
traits.trait_errors.TraitError: The 'out_file' trait of a MathsOutput instance must be a pathlike object or string representing an existing file, but a value of '/Users/lpzds1/Desktop/tidy/Test_dataset/HarvardOxford-Cortical_ROI_report/mean_HarvardOxford-Cortical_1901_WIPMB4TASKfMRI_singleechoTR2_e1.nii.gz' <class 'str'> was specified.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.7/Frameworks/Python.framework/Versions/3.7/lib/python3.7/tkinter/__init__.py", line 1705, in __call__
    return self.func(*args)
  File "fRAT_GUI.py", line 524, in Run_fRAT
    fRAT()
  File "/Users/lpzds1/python/brain_roi_analysis_tool/fRAT/fRAT.py", line 61, in fRAT
    brain_list = list(itertools.starmap(Utils.instance_method_handler, iterable))
  File "/Users/lpzds1/python/brain_roi_analysis_tool/fRAT/utils/utils.py", line 117, in instance_method_handler
    return getattr(argv[0], argv[1])(*argv[2:])
  File "/Users/lpzds1/python/brain_roi_analysis_tool/fRAT/utils/analysis.py", line 170, in run_analysis
    freesurfer_excluded_voxels = self.roi_flirt_transform()
  File "/Users/lpzds1/python/brain_roi_analysis_tool/fRAT/utils/analysis.py", line 183, in roi_flirt_transform
    '.nii.gz', self)
  File "/Users/lpzds1/python/brain_roi_analysis_tool/fRAT/utils/analysis.py", line 245, in fsl_functions
    fslfunc.run()
  File "/usr/local/lib/python3.7/site-packages/nipype/interfaces/base/core.py", line 421, in run
    outputs = self.aggregate_outputs(runtime)
  File "/usr/local/lib/python3.7/site-packages/nipype/interfaces/base/core.py", line 529, in aggregate_outputs
    raise FileNotFoundError(msg)
FileNotFoundError: No such file or directory '/Users/lpzds1/Desktop/tidy/Test_dataset/HarvardOxford-Cortical_ROI_report/mean_HarvardOxford-Cortical_1901_WIPMB4TASKfMRI_singleechoTR2_e1.nii.gz' for output 'out_file' of a MeanImage interface
```

**Comment by elliohow** [comment]
I've updated the fslpy version specified in the requirements.txt file. Can you update to the newest version to see if this fixes it?

**Comment by elliohow** [comment]
Fixed with eceba75

**Comment by elliohow** [comment]
Fixed with eceba75.
